<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdjParamDialog</class>
 <widget class="QDialog" name="AdjParamDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>校准与补偿</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabOffsets">
      <attribute name="title">
       <string>偏移补偿</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>组偏移 (Group Offset)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTreeWidget" name="treeGroupOffset">
            <column>
             <property name="text">
              <string>颜色/组</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QComboBox" name="comboGroupColor"/>
            </item>
            <item>
             <widget class="QComboBox" name="comboGroupGroup"/>
            </item>
            <item>
             <widget class="QComboBox" name="comboGroupDir"/>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxGroupOffset"/>
            </item>
            <item>
             <widget class="QPushButton" name="buttonGroupOffset">
              <property name="text">
               <string>应用组偏移</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>组内喷嘴偏移 (Jet-in-Group Offset)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTreeWidget" name="treeJetOffset">
            <column>
             <property name="text">
              <string>颜色/组/喷嘴</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QComboBox" name="comboJetColor"/>
            </item>
            <item>
             <widget class="QComboBox" name="comboJetGroup"/>
            </item>
            <item>
             <widget class="QComboBox" name="comboJetJet"/>
            </item>
            <item>
             <widget class="QComboBox" name="comboJetDir"/>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxJetOffset"/>
            </item>
            <item>
             <widget class="QPushButton" name="buttonJetOffset">
              <property name="text">
               <string>应用喷嘴偏移</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPrinting">
      <attribute name="title">
       <string>校准打印</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>打印参数</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>打印方向:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboPrintDir"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>校准DPI:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spinBoxAdjDpi"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>X运行速度 (mm/s):</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spinBoxRunSpeed"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>最大打印宽度 (mm):</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="spinBoxMaxWidth"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>最大打印高度 (mm):</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="spinBoxMaxHeight"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>X起始位置 (mm):</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDoubleSpinBox" name="spinBoxXPos"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>灰度 (bits):</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="spinBoxGrayBits"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>开始校准打印</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="buttonPrintStatus">
            <property name="text">
             <string>喷头状态测试</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="buttonPrintVertical">
            <property name="text">
             <string>垂直校准</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="buttonPrintStep">
            <property name="text">
             <string>步进校准</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="buttonPrintXOffsetGroup">
            <property name="text">
             <string>组间X向校准</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="buttonPrintXOffsetJet">
            <property name="text">
             <string>组内X向校准</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="buttonPrintBidirectional">
            <property name="text">
             <string>双向打印校准</string>
            </property>           
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonStopPrint">
         <property name="text">
          <string>停止打印</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="buttonLackJet">
       <property name="text">
        <string>缺嘴补偿...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonApplyAndClose">
       <property name="text">
        <string>应用并关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
